index.html

```diff
@@ -403,7 +264,7 @@
       </tr>
       <tr>
         <td>1</td>
-        <td>NFC Forum <dfn>well-known type</dfn> as defined in [[!NFC-RTD]]
+        <td>NFC Forum <dfn>well-known type</dfn> as defined in [[NFC-RTD]]
```

Comment:
We add this to be polite... and to remind other editors to never export these:

```suggestion
        <td>NFC Forum <dfn data-no-export="">well-known type</dfn> as defined in [[NFC-RTD]]
```